frontend/packages/dev-console/src/components/import/serverless/useUpdateKnScalingDefaultValues.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { useGetAutoscalerConfig } from './useGetAutoscalerConfig';
+
+export const setKnScalingDefaultValue = (initialValues, knScalingConfig) => {
+  const autoscalewindow =
+    knScalingConfig && knScalingConfig['stable-window'].match(/^[0-9]*|[a-zA-Z]*/g);
```

Comment:
it'll be nice to have in utils as used in on more place

Reply:
added it in util